mvpa2/base/externals.py

```diff
@@ -208,8 +211,9 @@ def __check_shogun(bottom_version, custom_versions=[]):
     if (ver in custom_versions) or (ver >= bottom_version):
         return True
     else:
-        raise ImportError, 'Version %s is smaller than needed %s' % \
-              (ver, bottom_version)
+        raise ImportError(
+            'Version %s is smaller than needed %s'
+            % (ver, bottom_version))
```

Comment:
PEP8: "The preferred place to break around a binary operator is after the operator, not before it. "